l3kernel/l3fp-functions.dtx

```diff
@@ -180,8 +180,10 @@
           \exp_args:No \clist_map_inline:nn { \tl_to_str:n {#3} }
             {
               \int_incr:N \l_@@_function_arg_int
-              \exp_args:Ne \@@_clear_variable:n
-                { _ \tex_romannumeral:D \l_@@_function_arg_int }
+              \exp_args:Ne \@@_clear_variable_aux:n
+                {
+                  \tl_to_str:e { _ \tex_romannumeral:D \l_@@_function_arg_int }
```

Comment:
Do we need `\tl_to_str:n` here? Isn't the internal path directly to a csname? The `_` must have catcode 11 whilst the roman numeral generation gives catcode 12.

Reply:
I'll change to `\c_underscore_str \tex_romannumeral:D \l_@@_function_arg_int`, to stick to the assumption that all first-args of internal forms `\@@_(new|set|clear)_(variable|function):...` are str.